docs/developer-guide/get-started/building-howto.md

```diff
@@ -122,11 +122,123 @@ Then, rebuild the image:
 ```bash
 sudo make image -j8 REBUILD_TOOLS=y REBUILD_PACKAGES=n CONFIG_FILE=./imageconfigs/edge-image.json
 ```
+### Add a new package
 
-### Update or Add Packages
+To add a new package you need to generate a `SPEC` file for the package which
+contains all required information for the build infrastructure to generate the
+`SRPM` and `RPM` for the package. There are a few steps involved to create
+a new package for the EMT.
 
-1. If a new package has to be released, follow these steps to ensure the package is available
-   in the artifactory:
+1. Defining the SPEC file and add it into the `/SPECS` directory
+1. Create the source archive and generate the sha256sum for the package
+1. Uploading the tar.gz package to the source package repository
+1. Add the SPEC file /SPEC directory and updating the `cgmanifest.json` file
+
```

Comment:
Can we add one more step here to say generate an image with this package included and test it?